server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -56,6 +87,69 @@ async function lowBatteryHeartbeat(coreId) {
 }
 
 describe('vitals.js integration tests: handleHeartbeat', () => {
+  describe('for a request with an incorrect API key', () => {
+    beforeEach(async () => {
+      await db.clearTables()
+
+      const client = await factories.clientDBFactory(db)
+      await locationDBFactory(db, {
+        locationid: testLocation1Id,
+        radarCoreId: radar_coreID,
+        clientId: client.id,
+      })
+
+      sandbox.stub(braveAlerter, 'startAlertSession')
+      sandbox.stub(braveAlerter, 'sendSingleAlert')
+      sandbox.spy(helpers, 'logSentry')
+      sandbox.spy(helpers, 'log')
+      sandbox.stub(db, 'logSensorsVital')
+    })
+
+    afterEach(async () => {
+      await db.clearTables()
+      sandbox.restore()
+    })
+
+    it('should return 200 for the valid request', async () => {
+      const request = {
+        coreid: radar_coreID,
+        data: `{"doorMissedMsg": 0, "doorLowBatt": true, "doorLastMessage": 1000, "resetReason": "NONE", "states":[]}`,
+        api_key: badpassword,
+      }
+      const response = await chai.request(server).post('/api/heartbeat').send(request)
```

Comment:
Is there a particular reason to define this request here? Is this different than `normalHeartbeatWithIncorrectAPIKey` in an important way?

Reply:
i think i got thrown off on how to get status 200 from it and then forgot to come back to update it to use the helpers!